tests/testthat/test_gridded.R

```diff
@@ -0,0 +1,101 @@
+# Test Gridded Data
+
+library(gcammaptools)
+context("Plotting gridded data")
+
+# load a gridded dataset
+prj <- rgcam::loadProject(system.file('sample-gcam-data','gcam-longform-sample.dat', package='gcammaptools'))
+co2grid <- rgcam::getQuery(prj, 'Cooling Degree Days', 'Reference')
+m <- map.rgn32.simple
+
+test_that("dataset has lat/lon values", {
+  expect_true("lon" %in% colnames(co2grid))
+  expect_true("lat" %in% colnames(co2grid))
+})
+
+test_that("plot_GCAM_grid plots with all defaults", {
```

Comment:
I suspect that none of these tests will catch errors in rendering the plots, since they don't occur until the `print.gg` method gets called.  What we probably need to do is to attempt to render these to temp files, or maybe look into the section on headless testing on Travis.  However, we can kick that can a little further down the road, for now.